Live_Tests_16_09/Live_Test_GUI_Automated.py

```
@@ -0,0 +1,272 @@
+            print("All tests passed! GUI is working correctly.")
+        else:
+            print(f"{total_tests - passed_tests} tests failed. Check issues above.")
+
+        # Close GUI if it was created (without showing it)
+        if self.gui and self.gui.root:
+            try:
+                # Don't call mainloop, just cleanup
+                self.gui.root.quit()
+                self.gui.root.destroy()
+                print("\nGUI cleaned up successfully")
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)